src/Chainweb/BlockHeader/Validation.hs

```diff
@@ -723,13 +723,20 @@ prop_block_chainId (ChainStep (ParentHeader p) b)
 -- Mutli chain inductive properties
 
 prop_block_epoch :: WebStep -> Bool
-prop_block_epoch (WebStep _as (ChainStep p b))
-    = _blockEpochStart b == epochStart p (_blockCreationTime b)
+prop_block_epoch (WebStep as (ChainStep p b))
+    = _blockEpochStart b <= EpochStartTime (_bct $ _blockCreationTime b)
+    && _blockEpochStart (_parentHeader p) <= _blockEpochStart b
+    && _blockEpochStart b == epochStart p as (_blockCreationTime b)
```

Comment:
This adds some sanity invariants for the epoch start value. A "sane" DA algorithm should guarantee these. Our current DA certainly assumes these properties.